deployment/Makefile

```diff
@@ -152,7 +152,7 @@ shell:
 	@echo "------------------------------------------------------------------"
 	@echo "Shelling in in production mode"
 	@echo "------------------------------------------------------------------"
-	@docker-compose -p $(PROJECT_ID) exec uwsgi /bin/bash
+	@docker-compose -p $(PROJECT_ID) exec devweb /bin/bash
```

Comment:
This looks wrong. L153, you must be in production mode.

Maybe better to create another make command for dev mode?

Reply:
Ah yes, I replace it incidentally. That's why we need peer review :). Thanks a lot.